www/js/lib/uiUtil.js

```diff
@@ -73,6 +73,19 @@ define(rqDef, function(settingsStore) {
                     resolve(false);
                 }
             });
+            // Enter key should accept default action
+            $("#alertModal").keydown(function(e){
```

Comment:
I wonder if keyup is more usual, i.e. you expect the event to occur after you have released the key? Main reason to use keydown is if you're trying to get in first before other event listeners.

I notice you have used jQuery here. Is this to match the style of similar keydown blocks we already have?

In line with #367, I now try to use only native DOM methods. But there's an argument for matching coding style of nearby blocks.

Otherwhise:
```
document.getElementById('alertModal').addEventListener('keyup', function (e) {
    ...
}
```

Reply:
Many thanks for this feedback.
The choice of keyup/keydown/keypress event was not really thought about, on my side. No problem to switch to keyup.

I used jQuery to match the style of the code above it, but I fully agree to switch to native DOM methods. I will change the code

Reply:
> The choice of keyup/keydown/keypress event was not really thought about, on my side. No problem to switch to keyup.

Use whatever seems most natural to you. I'm not very convinced by my own reasoning last night!